db/migrate/20230313195322_add_columns_state_to_preference_set.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnsStateToPreferenceSet < ActiveRecord::Migration[7.0]
+  def change
+    add_column :preference_sets, :table_state, :jsonb, default: {}
```

Comment:
make sure the `default` is a string has (`"{}"`). You can just do a `rake db:rollback STEP=1`, make the changes, then migrate again to fix

Reply:
I'll diggin deeper this weekend, but setting the default to a string json breaks it. I don't know why, but it works fine with the empty hash.

Reply:
hmmm, ok. I'll do some research on this to make sure. Maybe things have changed.

Reply:
The default here looks correct as: `{}`